PowerControl/Menu/MenuItemWithOptions.cs

```diff
@@ -13,7 +13,7 @@ public class MenuItemWithOptions : MenuItem
         public Func<string, string?>? ApplyValue { get; set; }
         public Func<string?>? ResetValue { get; set; }
 
-        public event Action<MenuItemWithOptions, String?, String> ValueChanged;
+        public event Action<MenuItemWithOptions, String?, String>? ValueChanged;
```

Comment:
This should not be needed, as specifically this is initalised in a construct of the class.

Reply:
Sure